.github/workflows/unit-tests.yml

```
@@ -27,7 +27,7 @@ jobs:
 
       - uses: arduino/cpp-test-action@main
         with:
-          runtime-path: extras/test/build/bin/testArduinoIoTCloud
+          runtime-paths: extras/test/build/bin/testArduinoIoTCloud
```

```suggestion
          runtime-paths: |
            - extras/test/build/bin/testArduinoIoTCloud
```

The data format of the `runtime-paths` input is different from `runtime-path`.

While the format of `runtime-path` only needed to be a simple string in order to accept a single path, `runtime-paths` must be able to accept multiple paths.

For this reason, it uses the format of a YAML sequence (i.e., array/list). But since GitHub Actions only allows inputs to be string type, this is a string containing a YAML document of sequence type, rather than an array in the workflow YAML document. This is the reason for the `|`.

---

Thank you @per1234 , I've got nothing to add :wink: